orbit/pkg/constant/constant.go

```diff
@@ -10,6 +10,9 @@ const (
 	// DefaultWorldReadableFileMode is the default file mode to apply to files
 	// that can be read by other processes.
 	DefaultWorldReadableFileMode = 0o644
+	// DefaultRestrictedReadFileMode is the default file mode to apply to files
+	// that can be read by other privileged processes.
+	DefaultRestrictedReadFileMode = 0o640
```

Comment:
AFAICS this would be used in Linux and macOS, what's the reason to change from `0o600` to `0o640`?

Reply:
It's a bit confusing because `DefaultFileMode` is more restrictive than `DefaultRestrictedReadFileMode`.

Reply:
You are right, it is better to use 0x600 here as that is a more restricted set of permissions. I totally missed that this constant was available.